MaxScaleProxyProblem/Program.cs

```diff
@@ -6,57 +6,66 @@ class Program
     {
         static void Main(string[] args)
         {
-            var context = new MaxScaleProxyDbContext();
-            int intTest = 1337;
-            long longTest = 1337;
-            DateTime dateTimeTest = DateTime.Now;
-            string stringTest = "test";
+			try
```

Comment:
I think you added one tab too much in here. I don't mind it since this is a repo for problem reproduction, but I think it's going to be more readable if there was one tab less.

Reply:
It looks like you're right.

The number of tabs looks fine in my local editor, but the problem is that with my change, the file now contains a mixture of tabs and spaces, so it can look different on different systems.